GNNGraphs/src/transform.jl

```diff
@@ -151,12 +151,14 @@ end
 
 """
     remove_edges(g::GNNGraph, edges_to_remove::AbstractVector{<:Integer})
+    remove_edges(g::GNNGraph, p::Float64=0.5)
```

Comment:
```suggestion
    remove_edges(g::GNNGraph, p=0.5)
```
no need to type restrict (especially since we don't to it in the signature).